Arc/Components/Progress/Viewcontrollers/TestCountDownViewController.swift

```diff
@@ -59,37 +65,44 @@ public class TestCountDownViewController: CustomViewController<TestCountDownView
     override public func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         self.customView.countLabel.font = UIFont(name: "Georgia-Italic", size: 96)
-        Animate().duration(0).delay(1.0).run { [weak self]
-            t in
-            self?.customView.countLabel.text = " 2 "
-            return true
-        }
-        Animate().duration(0).delay(2.0).run { [weak self]
-            t in
-            self?.customView.countLabel.text = " 1 "
-            return true
-        }
-        Animate().duration(0).delay(3.0).run { [weak self]
-            t in
-            
-            self?.countdownComplete()
-            return true
-        }
+        self.customView.countLabel.text = countdown3
+        self.countDown()
+    }
+    
+    func countDown() {
+        guard !self.isPaused else { return }
```

Comment:
Countdown can now be paused using the pause modal, and moved away from UIView animation for count down